### PR TITLE
driver: add missing MotionReply sub codes

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

Noticed they were missing while working on #435 and #436.

This will only improve the "human side" of `motoman_driver`. The error (sub) codes were processed correctly before this PR, but the human readable error message just converted them to "unknown" errors.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Perhaps @steviedale could (also) take a look?

File: motoman_driver/src/simple_message/motoman_motion_reply.cpp
Comment (marip8) [inline motoman_driver/src/simple_message/motoman_motion_reply.cpp]: If I remember correctly, PFL stands for power and force limiting and this error means that the driver can't be used when the robot is in PFL mode, correct? If so we might change this message to something more informative like:

```suggestion
    return "Power and force limiting (PFL) mode is active; disable PFL to execute trajectories with the ROS driver";
```

**Review by marip8 — DISMISSED** [review]
These changes look fine to me. I just had one comment about the PFL message we might want to address before merging.

File: motoman_driver/src/simple_message/motoman_motion_reply.cpp
Comment (gavanderhoorn) [reply motoman_driver/src/simple_message/motoman_motion_reply.cpp]: There are some nuances to this I believe.

Perhaps @EricMarcil can clarify?

Comment (EricMarcil) [reply motoman_driver/src/simple_message/motoman_motion_reply.cpp]: ROS can operate the robot with the PFL function enabled.  This message is when the PFL has detected a force that is above the allowed threshold (like someone pushing on the robot).  So we do not want to 'disable' the PFL but simply 'reset' the PFL, which is normally done by pressing the amber button on the robot B-link.
After that, ROS will need to reinitialize the trajectory mode (ROS_CMD_START_TRAJ_MODE = 200121) before sending a new trajectory from the robot current position.

Comment (marip8) [reply motoman_driver/src/simple_message/motoman_motion_reply.cpp]: So maybe a message like this would be better:

```suggestion
    return "Power and force limiting (PFL) has detected a force on the robot above the allowable threshold. Remove the obstruction and reset the PFL";
```

> After that, ROS will need to reinitialize the trajectory mode (ROS_CMD_START_TRAJ_MODE = 200121) before sending a new trajectory from the robot current position.

Is this something the user needs to do, or is it taken care of by the driver as a part of the trajectory execution action?

Comment (gavanderhoorn) [reply motoman_driver/src/simple_message/motoman_motion_reply.cpp]: New suggestion:

```suggestion
    return "Power and Force Limiting (PFL) was activated: "
      "please reset PFL on the robot, re-enable the driver (call the service) "
      "and send a new trajectory";
```

Comment (gavanderhoorn) [reply motoman_driver/src/simple_message/motoman_motion_reply.cpp]: > > After that, ROS will need to reinitialize the trajectory mode (ROS_CMD_START_TRAJ_MODE = 200121) before sending a new trajectory from the robot current position.
> 
> Is this something the user needs to do, or is it taken care of by the driver as a part of the trajectory execution action?

`motoman_driver` will try to enable trajectory mode upon reception of a new trajectory, but only if the driver has been "enabled" (by calling the `robot_enable` service) before.

It won't reset anything on the controller.

Comment (gavanderhoorn) [reply motoman_driver/src/simple_message/motoman_motion_reply.cpp]: Thanks for the input guys.

I've committed the new message for PFL.

**Comment by gavanderhoorn** [comment]
As I dismissed the review with my last commit, we're going to need another approval before this can be merged.

**Review by EricMarcil — APPROVED** [review]
@gavanderhoorn and @marip8 Looks good to me.  
Thanks

**Comment by gavanderhoorn** [comment]
Thanks for the reviews @marip8 and @EricMarcil :+1:

**Pull request merged**
